migrations/0000_next_puck.sql

```diff
@@ -1,6 +1,8 @@
 CREATE TABLE IF NOT EXISTS "bot_busters_match" (
 	"id" uuid PRIMARY KEY NOT NULL,
-	"room" json NOT NULL
+	"created_at" timestamp,
```

Comment:
Wait... why did the migrations get modified? Did you delete any old migration file?

Reply:
The migrations are modified because there are some changes to the db